src/app/package-json.js

```diff
@@ -168,13 +168,6 @@ const packageJSON = (current, context) => {
     'author': context.author,
     'license': context.licenseName,
 
-    'browserify': {
-      transform: [
-        'browserify-shim',
-        'browserify-versionify'
```

Comment:
Looks like both get included, though I don't know about ordering guarantees yet.
Also, if it's being configured through both the cli and the config it may be run twice.